### ISSUE TITLE
Error when trying to post a comment (SPAM-X issue)

### ISSUE THREAD

**Issue by eSilverStrike** [Geeklog-Core/geeklog#735]
On Geeklog.net the following error happens when I try to submit a comment (or a forum post) as anonymous user. It involves the SPAM-X plugin and probably relates to the fact that Geeklog.net is using PHP 5.3.29.

For a temporary fix on Geeklog.net I have deleted the BannedUsers.Examine.class.php file

```
Tue 17 Jan 12:32:08 2017 - 0.0.0.0 - E_WARNING(2) - preg_match() [<a href='function.preg-match'>function.preg-match</a>]: Compilation failed: nothing to repeat at offset 25 @ /path/plugins/spamx/BannedUsers.Examine.class.php line 63  
```

**Comment by eSilverStrike** [comment]
Okay I ran a few tests and it looks like it is a homepage with a bunch of question marks that causes the error. I confirmed this on my dev site. If you create a user with the homepage:

http://www.example.com/??????/

and then ban the user and try to submit content. You will generate the error.

**Comment by mystralkk** [comment]
Fixed with change set 7ab520b
